src/application/issuer.rs

```diff
@@ -217,20 +217,33 @@ impl Issuer {
         credential_schema: CredentialSchema,
         required_indices: Vec<u32>,
         nquads: Vec<String>,
-        revocation_list_did: &str,
-        revocation_list_id: &str,
+        revocation_list_did: Option<&str>,
+        revocation_list_id: Option<&str>,
         valid_until: Option<String>,
     ) -> Result<UnfinishedBbsCredential, Box<dyn Error>> {
-        let revocation_list_index_number = revocation_list_id
-            .parse::<usize>()
-            .map_err(|e| format!("Error parsing revocation_list_id: {}", e))?;
-
-        if revocation_list_index_number > MAX_REVOCATION_ENTRIES {
-            let error = format!(
+        let mut credential_status: Option<CredentialStatus> = None;
+        if revocation_list_id.is_some() {
```

Comment:
Maybe we could check for both, so we directly get an error, when only one is specified. Currently only specifying one, would continue silently.

```suggestion
        if revocation_list_id.is_some() || revocation_list_did.is_some() {
```